timesys/high-level-triggers.py

```diff
@@ -136,6 +136,8 @@
             'SI-13C3:PS-CV-2:BCKPLN',
             'SI-13C4:PS-CH:BCKPLN', 'SI-13C4:PS-CV:BCKPLN',
 
+            'SI-14SB:PS-CH-1:BCKPLN', 'SI-14SB:PS-CV-1:BCKPLN',
+            'SI-14SB:PS-CH-2:BCKPLN', 'SI-14SB:PS-CV-2:BCKPLN',
```

Comment:
@fernandohds564 what are these vertical correctors?

Reply:
I believe they don't exist, only horizontal correctors.

Reply:
@fernandohds564 , wiggler two correctors are only for the horizontal plane.

Reply:
ok, I removed them.